data-raw/stress_test_arguments.R

```diff
@@ -9,7 +9,7 @@ stress_test_arguments <- tibble::tribble(
             "discount_rate",    "double",   "0.07",                     NA,    "-0.01",      "0.1",
   "div_netprofit_prop_coef",    "double",      "1",                     NA,      "0.8",        "1",
                "shock_year",    "double",   "2030",                     NA,     "2025",     "2035",
-                     "term",    "double",      "2",                     NA,        "1",       "10",
```

Comment:
My understanding was that we allow an input of max 10 years and that the call of the Merton function automatically puts any term > 5 into the 5 year bucket. I think this should already be happening. We should probably still notify users in that case

Reply:
But why would we allow using values >10 if we cap them to 5 anyway?

Reply:
we assumed that institutes regularly have such terms in their loan books. In the end I don't really care and I think this is something we can revisit with some user feedback to make a good decision

Reply:
Or rather: if we decide we cap at years here already, then do you add a section that will turn any longer terms into 5 years when reading i the user input? I think it may be too much of a burden to ask them to manually change all numbers greater than 5 before running the software

Reply:
we do not cap year here already. This is the default values that are chosen.
The capping of values >5 (that can still happen as soon as setting terms on user level is activated) is independent of this and not affected by the PR.
However I would say the only reason one would not set the maximum fallback value for terms to 5 is if terms > 5 are used somewhere in an uncapped way that I am not aware of?

Reply:
They are not being used anywhere. It's fine for me to cap them as early as possible, but I would suggest to do so in a way that allows users to keep individual longer terms in the input data and rather bucket them in the code plus returning a message: "200 loans in the loan book had input term > 5 years. These were automatically set to 5 years by the credit risk model."